packages/node_modules/@ciscospark/plugin-people/src/people.js

```diff
@@ -51,9 +51,13 @@ const People = SparkPlugin.extend({
    *   // => success
    */
   get(person) {
+    if (!person) {
```

Comment:
@ianwremmel git "moved" this file instead of deleting it, so it appears that it isn't deleted